httpbp/client_middlewares.go

```diff
@@ -15,6 +16,8 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 
 	"github.com/reddit/baseplate.go/breakerbp"
+	"github.com/reddit/baseplate.go/faults"
+
```

Comment:
nit: don't add this extra blank line.

Reply:
`gofmt` is actually insisting this line is added -- maybe because the import beneath it has a comment above it? IMO we should just leave it as-is since `gofmt` recommends it.

Reply:
Ack. I assume this is OK to leave as-is since `goimports` recommends it? Or do you still want me to remove the line?

Reply:
that's in violation of https://github.com/reddit/baseplate.go/blob/master/Style.md#import-groups.

Reply:
My mistake. Fixed.